easybuild/easyconfigs/p/PFLOTRAN/PFLOTRAN-4.0-gofbf-2020a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'MakeCp'
+
+name = 'PFLOTRAN'
+version = '4.0'
+
+homepage = 'https://www.pflotran.org/'
+description = """
+PFLOTRAN is an open source, state-of-the-art massively parallel 
+subsurface flow and reactive transport code. PFLOTRAN solves a 
+system of generally nonlinear partial differential equations 
+describing multiphase, multicomponent and multiscale reactive flow 
+and transport in porous materials. The code is designed to run on 
+massively parallel computing architectures as well as workstations 
+and laptops.
+"""
+
+docurls = ['https://www.pflotran.org/documentation/']
+software_license = 'LicenseUnknown' #LGPL v2.1
+
+toolchain = {'name': 'gofbf', 'version': '2020a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://bitbucket.org/pflotran/%(namelower)s/get/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['5f1d9882505e145d6b57a03496167e5b38e013f6408a4a97359b834e2d7ac377']
+
+dependencies = [
+    ('PETSc', '3.17.1'),
+]
+
+skipsteps = ['configure']
+
+start_dir = 'src/pflotran'
+
+buildopts = 'pflotran LIBS="-L${EBROOTHDF5}/lib -lhdf5hl_fortran -lhdf5_hl -lhdf5_fortran -lhdf5 -lz -lgfortran"'
```

Comment:
Add a comment that HDF5 is loaded implicitly by PETSc or load explicitly HDF5 module, since its link on.

Reply:
Added HDF5 as direct dependency and clean the unnecessary library linking